plugin-server/src/common/services/quota-limiting.service.ts

```diff
@@ -0,0 +1,134 @@
+import { createRedisPool } from '~/utils/db/redis'
+import { TeamManager } from '~/utils/team-manager'
+
+import { PluginsServerConfig, RedisPool } from '../../types'
+import { LazyLoader } from '../../utils/lazy-loader'
+import { logger } from '../../utils/logger'
+
+// subset of resources that we care about in this service
+export type QuotaResource = 'events' | 'cdp_invocations'
```

Comment:
Are we specifically aiming to quota limit real-time destinations based on the product analytics `events`?